apk-builder.sh

```
@@ -0,0 +1,181 @@
+  }
+
+  cp "$OUTPUT_DIR/twa/app-release-signed.apk" "$OUTPUT_DIR/codex-web-twa.apk" 2>/dev/null
+  cd "$SCRIPT_DIR"
+  ok "APK built: $OUTPUT_DIR/codex-web-twa.apk"
+}
+
+# ── Method 2: AndroidJS (Node.js WebView) ──
+build_androidjs() {
+  info "Method 2: Building with AndroidJS (Node.js WebView wrapper)..."
+  local url="${1:-http://localhost:18923}"
+  local app_name="${2:-Codex Web}"
```

Shellcheck (0.11.0)

[warning] app_name appears unused. Verify use (or export if used externally).

(SC2034)